src/controllers/csi/provisioner/controller.go

```diff
@@ -214,9 +214,7 @@ func (provisioner *OneAgentProvisioner) updateAgentInstallation(ctx context.Cont
 
 	latestProcessModuleConfig = latestProcessModuleConfig.
 		AddHostGroup(dk.HostGroup()).
-		AddConnectionInfo(dk.Status.OneAgent.ConnectionInfoStatus, tenantToken).
-		AddTenantUUID(dynakubeMetadata.TenantUUID)
```

Comment:
`AddTenantUUID` is not used , should be removed from `dtclient.ProcessModuleConfig`

Reply:
done